app/assets/scripts/components/project/sec-panel/class-analytics-chart.js

```diff
@@ -25,10 +25,7 @@ const ChartContainer = styled.div`
 const ClassItem = styled.li`
   display: grid;
   grid-gap: 0.75rem;
-  grid-template-columns: ${({ bounds }) =>
-    bounds
-      ? '0.75rem minmax(10px, 1fr) 4rem 4rem'
-      : '0.75rem minmax(10px, 1fr) 2rem'};
+  grid-template-columns: 0.75rem minmax(10px, 1fr) 4rem 4rem;
```

Comment:
```suggestion
  grid-template-columns: ${({ totalArea }) =>
    totalArea
      ? '0.75rem minmax(10px, 1fr) 4rem 4rem'
      : '0.75rem minmax(10px, 1fr) 2rem'};
```
reinstate to ensure that the middle column only appears on the Class Distribution chart, not the F1 score chart